miner/bid_simulator.go

```diff
@@ -625,16 +628,39 @@ func (b *bidSimulator) simBid(interruptCh chan int32, bidRuntime *BidRuntime) {
 	// check if bid gas price is lower than min gas price
 	{
 		bidGasUsed := uint64(0)
-		bidGasFee := bidRuntime.env.state.GetBalance(consensus.SystemAddress)
+		bidGasFee := big.NewInt(0)
+
+		for i, receipt := range bidRuntime.env.receipts {
+			tx := bidRuntime.env.txs[i]
+			if !b.txpool.Has(tx.Hash()) {
+				bidGasUsed += receipt.GasUsed
+				effectiveTip, er := tx.EffectiveGasTip(bidRuntime.env.header.BaseFee)
+				if er != nil {
+					err = errors.New("failed to calculate effective tip")
+					return
+				}
+
+				if bidRuntime.env.header.BaseFee != nil {
+					effectiveTip.Add(effectiveTip, bidRuntime.env.header.BaseFee)
+				}
 
-		for _, receipt := range bidRuntime.env.receipts {
-			bidGasUsed += receipt.GasUsed
+				gasFee := new(big.Int).Mul(effectiveTip, new(big.Int).SetUint64(receipt.GasUsed))
+				bidGasFee.Add(bidGasFee, gasFee)
+
+				if tx.Type() == types.BlobTxType {
```

Comment:
don't add blobfees
otherwise 1 blob tx, gas price 1Gwei will be refused, because blobgasprice is low

Reply:
bidGasUsed does not add blobGasUsed